hvplot/converter.py

```diff
@@ -1617,7 +1699,7 @@ def _validate_kwds(self, kwds):
         # See e.g. alpha for Area plots with plotly:
         # https://github.com/holoviz/holoviews/issues/5226
         if self._backend_compat == 'bokeh':
-            combined_opts = combined_opts + self._style_options
+            combined_opts = combined_opts + self._style_options + self._interactivity_options
```

Comment:
I guess you have to update the comment above.

Reply:
what comment do you mean?

Reply:
```python
        # Only add the global styling options in the suggestions for bokeh
        # since they may not be supported by all the backends.
        # See e.g. alpha for Area plots with plotly:
        # https://github.com/holoviz/holoviews/issues/5226
        if self._backend_compat == 'bokeh':
            ...
```

Reply:
`Only add the global styling options ...` -> this is no longer true since you now added the interactivity options.